src/main/java/pb360/service/impl/GenerateRestApiServiceImpl.java

```diff
@@ -94,40 +86,202 @@ private MessageObject getCurrentDateTime() {
 		return date;
 	}
 
-
 	private void readJson(String link) {
 
-		JsonDataModel jsonData = new JsonDataModel();
-		String inputStr;
-		StringBuilder stringBuilder = new StringBuilder();
+		JSONParser parser = new JSONParser();
 
 		try {
-			File initialFile = new File(link);
-			InputStream targetStream = new FileInputStream(initialFile);
-			BufferedReader buffReader = new BufferedReader(new InputStreamReader(targetStream));
-			ObjectMapper mapper = new ObjectMapper();
-			mapper.enable(DeserializationFeature.ACCEPT_EMPTY_STRING_AS_NULL_OBJECT);
-			int n = 0;
-			while (( buffReader.readLine()) != null) {
-				stringBuilder.append(buffReader.readLine());
+
+			Object obj = parser.parse(new FileReader(link));
+
+			
+			JSONObject jsonObject = (JSONObject) obj;
+			//title
+			String title = (String) jsonObject.get("title");
+			// loop modelArray
+			//Model
```

Comment:
the code looks fine, but we need separate model to a small function, validator to a small function, that means we can easy to maintain in future